test/integration/kpatch-test

```diff
@@ -282,6 +283,14 @@ run_combined_test() {
 
 }
 
+# shellcheck disable=SC1091
+source /etc/os-release
+if [[ "${ID}" == "rhel" && "${VERSION_ID%%.*}" == "7" && "${VERSION_ID##*.}" -le "6" ]]; then
+	DYNDEBUG_ENABLED=0
+	echo "Dynamic debug is not supported on current distro, disabling:"
+	echo "${PRETTY_NAME}"
```

Comment:
Looks good, a minor nitpick on the preference, `${PRETTY_NAME}` can be inlined in to the warning, like  `echo "Dynamic debug is not supported on ${PRETTY_NAME}, disabling."`

Reply:
Thanks for the suggestion, I agree it will look better this way.